test/example.test.js

```diff
@@ -1,18 +1,185 @@
+    const expected = [{ id: 1, captured: 1, encountered: 1 }, { id: 2, captured: 0, encountered: 1 }];
+ 
+ //Act 
+ // Call the function you're testing and set the result to a const
+    setPokedex(newPokedex);
+    capturePokemon(1);
+    const stringyPokedex = localStorage.getItem('POKEDEX');
+    const actual = JSON.parse(stringyPokedex);
+
+ //Expect
+ // Make assertions about what is expected versus the actual result
+    expect.deepEqual(actual, expected);
+});
+
+test('This test should count all the total encounters and return that value as a number', (expect) => {
+    //Arrange
+    const newPokedex = [{ id: 1, captured: 0, encountered: 2 }, { id: 2, captured: 0, encountered: 3 }];
+ // Set up your arguments and expectations
+    const expected = 5;
+ 
+ //Act 
+ // Call the function you're testing and set the result to a const
+    setPokedex(newPokedex);
+    const actual = totalEncounters();
+
+ //Expect
+ // Make assertions about what is expected versus the actual result
+    expect.deepEqual(actual, expected);
+});
+
+test('This test should look at the localstorage and return the names of the items present.', (expect) => {
+    //Arrange
+    const newPokedex = [{ id: 1, captured: 0, encountered: 2 }, { id: 2, captured: 0, encountered: 3 }];
+ // Set up your arguments and expectations
+    const expected = ['Bulbasaur', 'Charmander'];
+ 
+ //Act 
+ // Call the function you're testing and set the result to a const
+    setPokedex(newPokedex);
+    const actual = getPokemonNames();
+
+ //Expect
+ // Make assertions about what is expected versus the actual result
+    expect.deepEqual(actual, expected);
+});
+
+test('This test should look at the localstorage and return the number of times each pokemon has been caught', (expect) => {
+    //Arrange
+    const newPokedex = [{ id: 1, captured: 2, encountered: 2 }, { id: 2, captured: 1, encountered: 3 }];
+ // Set up your arguments and expectations
+    const expected = [2, 1];
+ 
+ //Act 
+ // Call the function you're testing and set the result to a const
+    setPokedex(newPokedex);
+    const actual = getPokemonCaptures();
+
+ //Expect
+ // Make assertions about what is expected versus the actual result
+    expect.deepEqual(actual, expected);
+});
+
+test('This test should look at the localstorage and return the number of times each pokemon has been encountered', (expect) => {
+    //Arrange
+    const newPokedex = [{ id: 1, captured: 2, encountered: 2 }, { id: 2, captured: 1, encountered: 3 }];
+ // Set up your arguments and expectations
+    const expected = [2, 3];
+ 
+ //Act 
+ // Call the function you're testing and set the result to a const
+    setPokedex(newPokedex);
+    const actual = getPokemonEncounters();
+
+ //Expect
+ // Make assertions about what is expected versus the actual result
+    expect.deepEqual(actual, expected);
+});
+
+test('This test should look at the localstorage and return the colors of the items present.', (expect) => {
```

Comment:
Just wanted to say nice work with testing, and seemed like you tested for a lot more different functionality.   Looks great and can tell that you have a knack for TDD!